NuGetPackageVerifier.json

```diff
@@ -0,0 +1,134 @@
+{
+    "adx": { // Packages written by the ADX team and that ship on NuGet.org
+        "rules": [
+            "AssemblyHasDocumentFileRule",
+            "AssemblyHasVersionAttributesRule",
+            "AssemblyHasServicingAttributeRule",
+            "SatellitePackageRule",
+            "StrictSemanticVersionValidationRule"
+        ],
+        "packages": {
+            "dnx-clr-win-x64": {
+                "VERSION_INFORMATIONALVERSION": {
+                    "bin\\lib\\Microsoft.Dnx.DesignTimeHost\\Newtonsoft.Json.dll": "3rd party DLLs don't need this",
```

Comment:
The exclusions are printed here https://github.com/aspnet/DnxTools/blob/dev/src/NuGetPackageVerifier/Program.cs#L230. It prints it out that way and it is easier to copy as is.